regfilter/regfilter.py

```
@@ -251,21 +286,16 @@ async def on_message_edit(self, _prior, message):
+        regexs = await self.return_cache("regex")
+        if any(regex.search(content) for regex in regexs):
             names = await self.return_cache("names")
-            try:
-                name = random.choice(names)
-                await member.edit(nick = name, reason = "Filtered username")
-            except discord.HTTPException:
-                pass
-            except IndexError:
-                return
\ No newline at end of file
+            name = random.choice(names)
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)